cmd/cluster/cpd.go

```diff
@@ -182,9 +182,13 @@ func isSubnetRouteValid(awsClient aws.Client, subnetID string) (bool, error) {
 	}
 
 	for _, route := range describeRouteTablesOutput.RouteTables[0].Routes {
-		if *route.DestinationCidrBlock == "0.0.0.0/0" {
-			return true, nil
+		// Some routes don't use CIDR blocks as targets, so this needs to be checked
+		if route.DestinationCidrBlock != nil {
+			if *route.DestinationCidrBlock == "0.0.0.0/0" {
```

Comment:
Could we add this to the above if condition for a one-liner?

`if route.DestinationCidrBlock != nil && *route.DestinationCidrBlock == "0.0.0.0/0 {`

Reply:
I don't think that would work because if `*route.DestinationCidrBlock` is evaluated it'll be a nil pointer dereference and panic. This way the order is enforced and we ensure we don't have a nil pointer to start with